interpreter/Makefile

```diff
@@ -95,12 +95,10 @@ opam-release/%:
 	wget https://github.com/WebAssembly/spec/archive/opam-$*.zip
 	cp wasm.opam opam
 	echo 'url {' >> opam
```

Comment:
Nit: maybe also use `"` here for consistency.

Reply:
done :)